src/tribler-common/tribler_common/sentry_reporter/tests/test_sentry_scrubber.py

```diff
@@ -0,0 +1,253 @@
+import pytest
+
+from tribler_common.sentry_reporter.sentry_reporter import BREADCRUMBS, CONTEXTS, EXTRA, LOGENTRY, OS_ENVIRON, \
+    REPORTER, \
+    STACKTRACE, \
+    SYSINFO, \
+    SYS_ARGV
+from tribler_common.sentry_reporter.sentry_scrubber import SentryScrubber
+
+
+@pytest.fixture(name="scrubber")  # this workaround implemented only for pylint
+def fixture_scrubber():
+    return SentryScrubber()
+
+
+def test_patterns(scrubber):
+    # folders negative
+    assert not any(regex.search('') for regex in scrubber.re_folders)
+    assert not any(regex.search('some text') for regex in scrubber.re_folders)
+    assert not any(regex.search('/home//some/') for regex in scrubber.re_folders)
+
+    # folders positive
+    assert any(regex.search('/home/username/some/') for regex in scrubber.re_folders)
+    assert any(regex.search('/usr/local/path/') for regex in scrubber.re_folders)
+    assert any(regex.search('/Users/username/some/') for regex in scrubber.re_folders)
+    assert any(regex.search('/users/username/some/long_path') for regex in scrubber.re_folders)
+    assert any(regex.search('/home/username/some/') for regex in scrubber.re_folders)
+    assert any(regex.search('/data/media/username3/some/') for regex in scrubber.re_folders)
+    assert any(regex.search('WINNT\\Profiles\\username\\some') for regex in scrubber.re_folders)
+    assert any(regex.search('Documents and Settings\\username\\some') for regex in scrubber.re_folders)
+
+    # ip negative
+    assert not scrubber.re_ip.search('0.0.0')
```

Comment:
Maybe it is better to implement these tests as a list of arguments?

Reply:
Yes, maybe :)